pages/book.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { useEffect } from 'react'
+
+const Book: React.FC = () => {
+  useEffect(() => {
+    window.location.href =
+      'https://www.notion.so/garagescript/Table-of-Contents-a83980f81560429faca3821a9af8a5e2'
+  }, [])
+  return <></>
```

Comment:
Return `null` instead of `<></>`.
What is the point of this component do you plan to add content for `/book` or it is just a redirection?

Reply:
> Return `null` instead of `<></>`.
> What is the point of this component do you plan to add content for `/book` or it is just a redirection?

For now just redirection. Which is better approach? Returning null or <></> ?

Reply:
It don't even matter @Wolfy64 

This route will eventually be deprecated.

Reply:
@Wolfy64  Does it even matter if he returns `<></>` or `null` ?

Reply:
but still, what is a better approach? When to use null vs <></> ?

Reply:
It do not matter we could even return `false`, `true`, `undefined`, it's just a [common practice](https://stackoverflow.com/a/52223388/8279060).

Reply:
understood.